PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -60,10 +60,20 @@ internal struct PiwikUserDefaults {
     
     var clientId: String? {
         get {
-            return userDefaults.string(forKey: PiwikUserDefaults.Key.visitorID)
+            return userDefaults.string(forKey: PiwikUserDefaults.Key.clientID)
         }
         set {
-            userDefaults.setValue(newValue, forKey: PiwikUserDefaults.Key.visitorID)
+            userDefaults.setValue(newValue, forKey: PiwikUserDefaults.Key.clientID)
+            userDefaults.synchronize()
+        }
+    }
```

Comment:
If you change it this way, it will change the behavior of all old implementations. For all old Implementations the `clientId` was saved under the key `PiwikUserDefaults.Key.visitorID`. After this change, all existing installations will use the old `clientId` as their new `visitorId`. I thinks that's an issue.

Reply:
Good point. I've adjusted this in a9e9bf390f54fc714bbb49bbef760ace4d993b6a. I left my changes for using the correct "key name", but changing what value the key is using so it's using the old one. This should retain the existing functionality but be more clear in the future.

Reply:
Yeah, I think that's the best solution in this case. It's a bit of a pity that it got introduced in the first place. Thanks for changing it this way.